lib/ansible/modules/cloud/openstack/os_volume_snapshot.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# Copyright (c) 2016, Mario Santos <mario.rf.santos@gmail.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+
+DOCUMENTATION = '''
+---
+module: os_volume_snapshot
+short_description: Create/Delete Cinder Volume Snapshots
+extends_documentation_fragment: openstack
+version_added: "2.3"
```

Comment:
Just a note, we'll need to bump this to the correct version at the time this gets included.